java/ql/lib/semmle/code/java/security/UrlForwardQuery.qll

```diff
@@ -0,0 +1,35 @@
+/** Provides a taint-tracking configuration for reasoning about URL forwarding. */
+
+import java
+import semmle.code.java.security.UrlForward
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.security.PathSanitizer
```

Comment:
Since we now use parameterized dataflow configurations, the assumption that `*Query.qll` files will only be imported from queries is no longer true. So let's not indirectly expose all these imports:

```suggestion
private import semmle.code.java.security.UrlForward
private import semmle.code.java.dataflow.FlowSources
private import semmle.code.java.security.PathSanitizer
```

(Actually, the `*Query.qll` file isn't necessary and this could be in `UrlForward.qll` directly, but I leave that up to you in case you still want to keep things separated).

Reply:
Updated in https://github.com/github/codeql/pull/14854/commits/40c932a5f911c7ad5c1ca0cd5d517ee6081f4107.